### ISSUE TITLE
Implement TSOA and TSOP fields for ID3v2 tags

### ISSUE THREAD

**Issue by MBach** [MBach/Miam-Player#101]
In Miam-Player, there's already an option in settings to use user-defined articles to sort the Library, like "The". However, if a MP3 or FLAC file with ID3v2 Header has fields: 

- TSOP
> The 'Performer sort order' frame defines a string which should be
   used instead of the performer (TPE2) for sorting purposes.

- or TSOA
> The 'Album sort order' frame defines a string which should be used
   instead of the album name (TALB) for sorting purposes. E.g. an album
   named "A Soundtrack" might preferably be sorted as "Soundtrack".

- Just for the record, there's also TSOT
> The 'Title sort order' frame defines a string which should be used
   instead of the title (TIT2) for sorting purposes.

Miam-Player should automatically consider that one wants those tracks to be well sorted. This field is unusual, so the software shouldn't let one to modify the behavior if options: if TSOP is written in header, always sort, no matter what. TSOP must take precedence:
- The option is unchecked in options:
![sorting_library](https://cloud.githubusercontent.com/assets/1532657/14066613/2233d4c8-f44f-11e5-9242-158f86a30f37.png)
- Some tracks have TSOP field set, like "Doors"

Then "The Doors" should be listed under "D", not "T"

In the Tag Editor, some custom fields might be displayed (with context menu, not in the default view) if one wants to add or remove them. This kind of modifications should be described in another Issue once this one is done.
